### DISCUSSION TITLE
Does QueCleaner.bash delete torrent files.

### DISCUSSION THREAD

**Discussion by DwarfJockey** [RandomNinjaAtk/docker-sonarr-extended#9] [Q&A] [answered]
Got a situation with a possibly obvious answer. I have some torrents that I'm getting HnR's on that keep disappearing from qBittorrent randomly. I re-download them and the same torrents get deleted. Check the logs of qBittorrent, Sonarr, qbit_manage, ect and I don't see any mention of said files deleting. I'm now suspect that QueCleaner is deleting them. Logs say that they are being removed from the que, but I don't know if they are triggering the script because the files went missing or if the files are missing because of the script. I probably sound like an idiot, but I guess I'm asking if "remove from que" means literally deleting it from qbittorrent or just removing stuck files from Sonarr's que?

**Comment by RandomNinjaAtk** [comment, marked as answer]
So queue cleaner, uses Sonarr's built in function to remove it from the queue. It should remove/delete the files in the download client and also remove it from the download client.

This is intentional to remove the need for user intervention for downloads that failed to automatically import, because the script will trigger Sonarr to perform a new search to try a new alternate download, that could likely be imported automatically. 

If you don't desire to have this level of automation, you could disable it. I'd prefer not to have to do anything manually, so that is why I created a script to handle it automatically.

**Reply by DwarfJockey** [reply]
Cool, thanks. I'm fine with disabling it. It's certainly a cool feature. For my use case I'll let qbit_manage handle my torrents as I need to guarantee the files seed for a minimum amount of time.
